api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -86,7 +86,7 @@ impl BrokerRpcApiServer for RpcServerImpl {
 		refund_parameters: RefundParametersRpc,
 		dca_parameters: Option<DcaParameters>,
 	) -> RpcResult<SwapDepositAddress> {
-		Ok(self
+		let swap_deposit_address = self
```

Comment:
would be nice to log some of the input parameters too, this will give us a starting timestamp and make it maybe easier to correlate the logs.

ideally we would have some unique request id here to make it very simple